packages/discovery-provider/ddl/migrations/0071_album_price_history_cascade_delete.sql

```diff
@@ -0,0 +1,12 @@
+begin;
+
+-- drop the existing foreign key constraint
+alter table album_price_history drop constraint if exists blocknumber_fkey;
+alter table album_price_history drop constraint if exists album_price_history_blocknumber_fkey;
+
+-- add the foreign key constraint with on delete cascade
+alter table album_price_history
```

Comment:
tbh i do not know this stuff very well. but do we need to do what `track_price_history` is doing with the if statement here?
https://github.com/AudiusProject/audius-protocol/blob/93a83f81d26455b4ea64426855fd9538e58f1c07/packages/discovery-provider/ddl/migrations/0025_add_track_price_history_fkey.sql#L6